src/main/java/net/mcreator/ui/dialogs/wysiwyg/LabelDialog.java

```diff
@@ -83,6 +84,13 @@ public LabelDialog(WYSIWYGEditor editor, @Nullable Label label) {
 		}
 
 		options.add(PanelUtils.join(FlowLayout.LEFT, L10N.label("dialog.gui.label_text_color"), cola));
+
+		final JComboBox<IAnchorableElement.AnchorPoint> anchor = new JComboBox<>(IAnchorableElement.AnchorPoint.values());
+		if (!editor.isNotOverlayType) {

```

Comment:
Is there a reason to add an overlay restriction? Anchor points can for sure be added to custom GUIs, so it should be done in this PR too (but I can understand to keep the other components for another PR)

Reply:
I didn't think it would make sense in custom GUIs. If it's ok I'd like to limit this to overlays and the 3 components that are usable there.

Reply:
GUIs are similar to overlays, so I personally think this would make sense to have the feature for them too as it can help users with some cases. However, we can ask @KlemenDEV his opinion about the question.

Reply:
Usually, GUIs anchor the way we do now, practically all GUIs in vanilla too.

So for now I would only implement this for overlays

Reply:
LGTM